projects/storefrontstyles/scss/components/myaccount/_my-coupons-dialog.scss

```diff
@@ -0,0 +1,74 @@
+%cx-coupon-dialog {
+  .cx-dialog-item {
+    padding: 0 1rem 0 0;
+  }
+  .cx-coupon-name {
+    font-weight: var(--cx-font-weight, var(--cx-font-weight-bold, 700));
```

Comment:
We're not using CSS variables for fonts anymore, we only use the `@include type()`
 **mixin** for fonts, you can use this reference table below:
![Screen Shot 2019-09-27 at 10 59 00 AM](https://user-images.githubusercontent.com/15113219/71752383-dee28200-2e4c-11ea-8d93-08dd27104fe8.png)

Reply:
updated